pkg/logs/config/tests/complete/conf.d/integration.yaml

```diff
@@ -9,4 +9,14 @@ logs:
     service: nginx
     source: nginx
     sourcecategory: http_access
-    tags: env:prod
```

Comment:
I would have left this integration file the same and created a new one for your new feature so that we ensure we did not break the current behaviour.

Reply:
Fixed